blog/BCHN-Financial-Report-2020-10-20.md

```diff
@@ -0,0 +1,218 @@
+
+* * *
+
+### Links
+
+Previous financial overviews:
+
+*   [Financial overview in report of 14 March 2020](https://read.cash/@freetrader/bchn-lead-maintainer-report-2020-03-14-84f2992d#financial-overview)
+
+*   [Financial overview in report of 31 March 2020](https://read.cash/@freetrader/bchn-lead-maintainer-report-2020-03-31-1e0a4c39#financial-overview)
+
+*   [Financial overview in report of 15 April 2020](https://read.cash/@freetrader/bchn-lead-maintainer-report-2020-04-15-449fc115#financial-overview)
+
+*   [BCHN Financial Report 2020-06-03](https://read.cash/@bitcoincashnode/bchn-financial-report-2020-06-03-d2a0232f) (3 June 2020)
+
+*   [BCHN Financial Report 2020-07-07](https://read.cash/@bitcoincashnode/bchn-financial-report-2020-07-07-d04c7e65) (7 July 2020)
+
+*   [BCHN Financial Report 2020-09-03](https://read.cash/@bitcoincashnode/bchn-financial-report-2020-09-03-a477d4e2) (20 October 2020)
+
+Flipstarter funding proposal:
+
+*   [English, v1.1](https://gitlab.com/bitcoin-cash-node/bchn-project-management/bchn-pm-public/-/raw/master/finance/funding/campaigns/2020-q2-flipstarter/pdf/Bitcoin_Cash_Node_Flipstarter_Funding_Proposal_v1_1.pdf) (PDF)
+
+*   [Chinese, v1.1](https://gitlab.com/bitcoin-cash-node/bchn-project-management/bchn-pm-public/-/blob/master/finance/funding/campaigns/2020-q2-flipstarter/pdf/Bitcoin_Cash_Node_Flipstarter_Funding_Proposal_v1_1_CN.pdf) (PDF)
+
+*   [Spanish, 1.1](https://gitlab.com/bitcoin-cash-node/bchn-project-management/bchn-pm-public/-/blob/master/finance/funding/campaigns/2020-q2-flipstarter/proposal/proposal-es.mediawiki) (mediawiki)
+
+**Getting in touch with BCHN:**
+
+*   Website: [https://bitcoincashnode.org](https://bitcoincashnode.org)
+
+*   Development & support Slack chat invite link:
+
+    [https://join.slack.com/t/bitcoincashnode/shared_invite/zt-egg3c36d-2cglIrKcbnGpIQFaKFzCWA](https://join.slack.com/t/bitcoincashnode/shared_invite/zt-egg3c36d-2cglIrKcbnGpIQFaKFzCWA)
+
+*   Telegram: https://t.me/bitcoincashnode (there is a bridge channel to our Slack)
+
+*   IRC channel: Join [#bchnode](/search?q=%23bchnode) on Freenode (we see messages on our Slack via an IRC bridge channel)
+
+*   Logs of our development Slack: [http://logs.bchnode.org/](http://logs.bchnode.org/)
+
+*   Main development repository on GitLab:
+
+    [https://gitlab.com/bitcoin-cash-node/bitcoin-cash-node](https://gitlab.com/bitcoin-cash-node/bitcoin-cash-node)
+
+*   Easy download link via our website:
+
+    [https://bitcoincashnode.org/download.html](https://bitcoincashnode.org/download.html)
+
+*   Full releases on Github:
+
+    [https://github.com/bitcoin-cash-node/bitcoin-cash-node/releases](https://github.com/bitcoin-cash-node/bitcoin-cash-node/releases)
+
+*   Donation address: [https://bitcoincashnode.org/#donate](https://bitcoincashnode.org/#donate)
+
+Follow us on Twitter: [https://twitter.com/bitcoincashnode](https://twitter.com/bitcoincashnode)
```

Comment:
I'm guessing this is supposed to be a bullet point also. If not, it probably should be because it looks weird.

Reply:
Fixed in #139